pandas/tests/arrays/boolean/test_logical.py

```diff
@@ -225,3 +230,11 @@ def test_no_masked_assumptions(self, other, all_logical_operators):
             result = getattr(a, all_logical_operators)(other)
             expected = getattr(b, all_logical_operators)(other)
             tm.assert_extension_array_equal(result, expected)
+
+
+@pytest.mark.parametrize("operation", [kleene_or, kleene_xor, kleene_and])
+def test_error_both_scalar(operation):
+    msg = "Either `left` or `right` need to be a np.ndarray."
```

Comment:
should escape the period. (or re.escape the string)

Reply:
escaped